src/background.ts

```diff
@@ -52,6 +52,7 @@ import * as Proxy from "@src/lib/proxy"
     R,
     perf,
     meta,
+    tabs: tabsProxy,
```

Comment:
I wonder if this should be attached to the window instead of `tri`. It's not good to litter the window object, but it'd save typing 4 characters, which would be nice for interactive use and would make `alias` commands in tridactylrcs more readable.

Reply:
Are you trying to avoid e.g. `tri.tabs[2].tri`?

We should probably document the object somewhere. Maybe `:jsb`? What was your use case for it?

Reply:
>Are you trying to avoid e.g. tri.tabs[2].tri?

Not specifically, it's just that in practice I found it slightly annoying to have to type the first `tri.` and would have preferred to have direct access to the `tabs` object. I doubt I'll have to type it a lot though, so it probably doesn't matter much.

>We should probably document the object somewhere. Maybe :jsb? 

Ha, you're right, forgot to document it. Will do in ~12 hours.

>What was your use case for it?

Basically I had a bunch of bandcamp user profile tabs open and wanted to close the ones that had less than 10 albums in common with mine, so I thought I'd do something like
```javascript
:jsb browser.tabs.query({url: "bandcamp.com"})
    .then(tabs =>
        tabs.forEach(tab => 
            tri.tabs[tab.id].tri.excmds.js("parseInt(document.querySelectorAll('li.genre-wrapper')[0].innerText.match(/([0-9]*) items? in common/)[1])")
                .then(count => { if (count < 10) { browser.tabs.remove(tab.id) } })
        )
    )
```
Without the newlines, of course :)

Reply:
>We should probably document the object somewhere.

Done.